src/main/java/com/powsybl/openloadflow/ac/solver/AcSolverParameters.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * Copyright (c) 2024, Artelys (http://www.artelys.com)
  * This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Isn't there a way to add a copyright instead of replacing it ?
Same for author below.